packages/xrpc/src/types.ts

```diff
@@ -24,7 +24,9 @@ export const errorResponseBody = z.object({
 export type ErrorResponseBody = z.infer<typeof errorResponseBody>
 
 export enum ResponseType {
+  // Network issue, unable to get response from the server.
```

Comment:
JS Doc will have better IDE integration (tooltip will contain message on hover).

```suggestion
  /**
   * Network issue, unable to get response from the server.
   */
```